src/Equinox.Projection.Kafka/Kafka.fs

```diff
@@ -317,6 +330,22 @@ type KafkaConsumer private (log : ILogger, consumer : Consumer<string, string>,
             ConsumerBuilder<_,_>(config.Kvps)
                 .SetLogHandler(fun _c m -> log.Information("consumer_info|{message} level={level} name={name} facility={facility}", m.Message, m.Level, m.Name, m.Facility))
                 .SetErrorHandler(fun _c e -> log.Error("Consuming... Error reason={reason} code={code} broker={isBrokerError}", e.Reason, e.Code, e.IsBrokerError))
+                .SetStatisticsHandler(fun _c json -> 
+                    // Stats format: https://github.com/edenhill/librdkafka/blob/master/STATISTICS.md
+                    let stats = JToken.Parse json
+                    (stats.Item "topics").Children()
+                    |> Seq.choose(fun t -> 
+                        if t.HasValues then Some t.First else None)
+                    |> Seq.filter(fun t -> config.topics |> Seq.exists(fun top -> top = ((t.Item "topic").ToString())))
+                    |> Seq.iter (fun topicMetric ->                     
+                        let topic =( topicMetric.Item "topic").ToString()
+                        let metrics = 
+                            (topicMetric.Item "partitions").Children()
+                            |> Seq.choose(fun t -> 
+                                if t.HasValues then Some t.First else None)
+                            |> Seq.map(fun m -> m.ToObject<KafkaPartitionMetrics>())
+                            |> Seq.filter(fun m -> m.partition <> -1)                        
+                        log.Information("consumer stats reporting {topic} | {stats}", topic, metrics)))
```

Comment:
using `{@token}` in the expression will make it be a json structure when rendered to a json log, in addition to looking prettier on the console or in https://getseq.net